src/github-api-helper.ts

```diff
@@ -134,7 +134,7 @@ async function downloadArchive(
     repo: repo,
     ref: commit || ref
   })
-  if (response.status != 302) {
+  if (response.status != 200) {
```

Comment:
could we accept both 200 and 302? And: could the commit message please provide context for this change?

Reply:
This was edited on mobile phone to see if it can make CI pass, but unfortunately seems running CI requires manual approval. I'll do a proper amend commit on my computer once I have time.